### PR TITLE
refactor(messages): remove modal & replace with icons

### PR DESCRIPTION
**Pull request by NuttyShrimp**
As discussed in #426 

**Pull Request Description**

_Please include a general description of the changes made, why they were necessary, 
and any other general things to note._

**Pull Request Checklist**:
* [x] Have you followed the guidelines in our Contributing document and Code of Conduct?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/project-error/npwd/pulls) for the same update/change?
* [x] Have you built and tested NPWD in-game after the relevant change?

### CODE REVIEW

**Comment by itschip** [comment]
Do you have an image on how this looks?

**Comment by NuttyShrimp** [comment]
Yes. I used a smaller fontsize than I original showed in the issue because you mentioned it.
Only 'issue' I see is that the ripple around the button is not a perfect circle but I think that's more a MUI issue
image

**Comment by itschip** [comment]
While we're one this, you can probably just remove the whole modal and the `MoreVert` icon stuff

**Comment by NuttyShrimp** [comment]
So, I am currently replacing the modal but I can already see people accidentally clicking the wrong button when trying to call someone and removing there conversation.
So do I keep the modal and replace the contents to act like a confirm box to prevent users from accidentally removing there conversation?

**Comment by itschip** [comment]
> So, I am currently replacing the modal but I can already see people accidentally clicking the wrong button when trying to call someone and removing there conversation. So do I keep the modal and replace the contents to act like a confirm box to prevent users from accidentally removing there conversation?

There won't be any way to 'accidentally clicking the wrong button' when there only will be one. Well, two if you don't have the person as a contact, but the group actions modal should not exist at all. Nor should the 'MoreVert' icon button either.

File: phone/src/apps/messages/components/modal/MessageModal.tsx [hunk 1]
```diff
@@ -176,11 +151,11 @@ export const MessageModal = () => {
             </Typography>
             <Tooltip
               classes={{ tooltip: classes.tooltip }}
-              title={t('MESSAGES.ACTIONS_TITLE')}
-              placement="left"
+              title={`${t('DIALER.NAVBAR_DIAL')} ${targetNumber}`}
```
Comment (itschip) [inline phone/src/apps/messages/components/modal/MessageModal.tsx:154]: Let's use the generic translation key here :)

```suggestion
              title={`${t('GENERIC.CALL')} ${targetNumber}`}
```

**Review by itschip — CHANGES_REQUESTED** [review]

File: phone/src/apps/messages/components/modal/MessageModal.tsx (hunk 1 above)
Comment (NuttyShrimp) [reply phone/src/apps/messages/components/modal/MessageModal.tsx:154]: Ah I was searching for it but couldn't find it.

**Review by itschip — APPROVED** [review]

**Pull request merged**